packages/components/src/highlight-cards/annual-highlight-cards.tsx

```diff
@@ -42,33 +44,41 @@ export default function AnnualHighlightCards( {
 
 	return (
 		<div className={ classNames( 'highlight-cards', className ?? null ) }>
-			{ header }
+			<div className="highlight-year-navigation">
+				{ header }
+				{ navigation && navigation }
```

Comment:
```suggestion
				{ navigation }
```

Could the `undefined` rendering check be ignored?

Reply:
Yes, of course, thanks for catching this.